.github/workflows/release.yml.template

```
@@ -58,7 +63,7 @@ jobs:
       -
         name: Annotate Wasm module
         run: |
-          {{ "./kwctl annotate -m ${{ env.METADATA_FILE }} -o policy-annotated.wasm target/wasm32-unknown-unknown/release/${WASM_BINARY_NAME}.wasm" }}
+          {{ "kwctl annotate -m ${{ env.METADATA_FILE }} -o policy-annotated.wasm target/wasm32-unknown-unknown/release/${WASM_BINARY_NAME}.wasm" }}
```

I think this should work as fine:

```suggestion
          kwctl annotate -m ${{ env.METADATA_FILE }} -o policy-annotated.wasm target/wasm32-unknown-unknown/release/${WASM_BINARY_NAME}.wasm
```

---

I tried to apply your suggestion, but cargo generate breaks. He's confused by the GH action template `${{ ... }}`, it assumes this template belongs to him and fails